es-app/src/guis/GuiMenu.cpp

```diff
@@ -500,6 +503,8 @@ void GuiMenu::openQuitMenu()
 	row.makeAcceptInputHandler([window] {
 		window->pushGui(new GuiMsgBox(window, "REALLY RESTART?", "YES",
 			[] {
+			Scripting::fireEvent("quit");
+			Scripting::fireEvent("restart");
```

Comment:
on your wiki you have 

```reboot | on system reboot (also calls quit first)```

shouldn't this line be ``` Scripting::fireEvent("reboot"); ```

Reply:
fixed